packages/web/src/common/store/pages/collection/selectors.ts

```diff
@@ -17,15 +17,23 @@ export const getCollectionStatus = (state: CommonState) =>
 export const getSmartCollectionVariant = (state: CommonState) =>
   state.pages.collection.smartCollectionVariant
 
-export const getCollection = (state: CommonState) => {
+export const getCollection = (state: CommonState, params?: { id: ID }) => {
   const smartCollectionVariant = getSmartCollectionVariant(state)
   if (smartCollectionVariant) {
     return getSmartCollection(state, { variant: smartCollectionVariant })
   }
-  return getCachedCollection(state, { uid: getCollectionUid(state) })
+
+  const config = params?.id
+    ? { id: params?.id }
```

Comment:
I wouldn't think you'd need params*?*.id the second time right?

Reply:
yup